runtime/compiler/z/env/J9CPU.cpp

```diff
@@ -52,11 +52,17 @@ J9::Z::CPU::detectRelocatable(OMRPortLibrary * const omrPortLib)
    omrsysinfo_get_processor_description(&processorDescription);
 
    if (processorDescription.processor > OMR_PROCESSOR_S390_Z10)
-      { 
+      {
       processorDescription.processor = OMR_PROCESSOR_S390_Z10;
       processorDescription.physicalProcessor = OMR_PROCESSOR_S390_Z10;
       }
 
+   const uint32_t disabledFeatures [] = { OMR_FEATURE_S390_TE };
+   for (size_t i = 0; i < sizeof(disabledFeatures)/sizeof(uint32_t); i++)
+      {
+      omrsysinfo_processor_set_feature(&portableProcessorDescription, disabledFeatures[i], FALSE);
```

Comment:
Just realized this won't build -_-

Reply:
Updated, see https://github.com/eclipse-openj9/openj9/compare/9cc24666b8d48ae934ff9a9f822e72c2ed6864b1..b86eb7717a858a50437edbf29364ed3b0d9c9d26